frontend/src/metabase/query_builder/actions.js

```diff
@@ -729,6 +734,17 @@ export const setParameterValue = createAction(
   },
 );
 
+export const SOFT_RELOAD_CARD = "metabase/qb/SOFT_RELOAD_CARD";
```

Comment:
kind of hate doing this so am open to alternatives. the issue right now is that a `moderation_review` POST mutates the associated card object and it may mutate more than just the list of `moderation_reviews` on the card, as a review can potentially resolve associated `moderation_requests` as well.

calling this a "soft" reload as it GETs the card object but does not re-run the query as the below `RELOAD_CARD` action does

Reply:
Sounds reasonable, I would add a comment in code with the explanation

Reply:
added